script/test-setup.sh

```diff
@@ -0,0 +1,2 @@
+# setup a topic for integration testing
+vagrant ssh kafka -c '/opt/kafka_2.11-0.10.1.0/bin/kafka-topics.sh --zookeeper localhost:2181 --create --partitions 32 --replication-factor 1 --topic kaffe-test --config retention.ms=1209600000'
```

Comment:
To make it generic you could say something like the end to end tests expect to be able to reach kafka at the hostname/port of kafka:9092 and have a topic of "kaffe-test" provisioned with 32 partitions.